### ISSUE TITLE
BUG: CORS error when editing posting

### ISSUE THREAD

**Issue by eigenfoo** [eigenfoo/euphoria#103]
```
Access to fetch at 'http://199.98.27.117:8080/api/posting' from origin 'http://199.98.27.117:8000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Comment by luster** [comment]
If you go through nginx you shouldn’t have this problem. Port change (calling 8080 from 8000) appears to be different origin

**Comment by eigenfoo** [comment]
@luster it looks like this is only a problem with making PUT or DELETE requests: our GET and POST requests work just fine. We're trying [this nginx config](https://enable-cors.org/server_nginx.html). Would that help?

**Comment by eigenfoo** [comment]
Closed by #127
